.github/workflows/release.yml

```diff
@@ -404,19 +404,19 @@ jobs:
           role-to-assume: ${{ secrets.AWS_ROLE_MARKETPLACE }}
 
       - name: Publish to AWS Marketplace
-        uses: nginxinc/aws-marketplace-publish@108e752101152582ed409c5faed859a891e0d7aa # v1.0.7
+        uses: nginx/aws-marketplace-publish@accf7b4c725796b744f2ee27acc2488d76f63d32 # v1.0.8
```

Comment:
this is both an org change and a patch upgrade. Is this expected here?

Reply:
yes, we've released new versions of all the projects after the migration

Reply:
now it's just changing `nginx`